scripts/linters/custom_eslint_checks/rules/newline-between-testcases.js

```diff
@@ -0,0 +1,61 @@
+// Copyright 2021 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview Lint check to ensure that there is exactly one line break
+ * after each test cases.
+ */
+
+'use strict';
+
+module.exports = {
+  meta: {
+    type: 'layout',
+    docs: {
+      description: 'There should be a single newline between test cases.',
+      category: 'Stylistic Issues',
+      recommended: true
+    },
+    fixable: null,
+    schema: [],
+    messages: {
+      line: 'There should be a single newline break before it()'
+    }
+  },
+
+  create: function(context) {
+    const sourceCode = context.getSourceCode();
+    const lines = sourceCode.lines;
+
+    var checkLineBreak = function(node) {
+      var line = node.loc.end.line;
+      if (lines[line].trim() !== '' && !lines[line].trim().startsWith('}') ||
+        lines[line] === '' && lines[line + 1] === '') {
```

Comment:
Instead of parsing the entire line with so many check can we do this:

(token before the function call).lineno + 1 == callExpression.endlinenumber?

Reply:
@DubeySandeep Thanks for suggestion . I tried this logic but it is giving me error when we have comment between testcases.

Reply:
You have to update the functionality to ignore comment lines (note you can check whether the token is a token comment or anything else). Is that helpful?